### DISCUSSION TITLE
rfcomm status

### DISCUSSION THREAD

**Discussion by anneranch** [blueman-project/blueman#2152] [Q&A]
blueman:
BlueZ:
Distribution:
Desktop environment:

* [x] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

<!--

rfcomm -a output and status message in Bluetooth manager do not
agree.

Connection Failed: Failed to get rfcomm channel

nov25-1@nov251-desktop:~$ rfcomm -a
rfcomm0: 00:15:83:15:A2:CB -> 98:D3:31:F8:39:33 channel 1 connected [reuse-dlc release-on-hup tty-attached]
nov25-1@nov251-desktop:~$ 

The adapter light is on steady , hence device is connected.

->

**Comment by anneranch** [comment]
Bluetooth  manager app  and rfcomm -a data do not agree 

nov25-1@nov251-desktop:~$ rfcomm -a
rfcomm0: 00:15:83:15:A2:CB -> 98:D3:31:F8:39:33 channel 1 connected [reuse-dlc release-on-hup tty-attached]


Connection Failed: Failed to get rfcomm channel

The device light is on, hence the device is indeed connected.

**Comment by infirit** [comment]
rfcomm is not provided by blueman, it's a deprecated utility by the BlueZ project.

**Reply by anneranch** [reply]
Is it customary on this "forum" to provide reasonable discussion or not ? 
I posted an observation that bluetooth 
 manager and rfcomm states do not match. 

I did not ask what rfcomm is....
I guess mismatch of two tools is not important.

Irregadles - I did not expect the discussion to be closed.

**Reply by infirit** [reply]
Keep the discussion to the earlier thread. You seem to be confused about things and spreading it over multiple threads isn't helpful.
